virttest/guest_agent.py

```diff
@@ -730,3 +725,86 @@ def fsthaw(self, check_status=True):
                     raise
             return ret
         return -1
+
+    @error_context.context_aware
+    def file_open(self, path, mode=None):
+        """
+        Open a guest file
+        :param path: full path to the file in the guest to open.
+        :param mode: optional open mode, "r" is the default value.
```

Comment:
What is the expected type of mode? string?

Reply:
Hi, the type of "mode" is string,is there any issue for python3 ?